velociraptor/observations/objects.py

```diff
@@ -222,6 +227,9 @@ class ObservationalData(object):
     bibcode: str
     # redshift that the data is at
     redshift: float
+    # optional, redshift upper and lower bounds for plotting
+    redshift_lower: float
+    redshift_upper: float
```

Comment:
Currently, these two attributes are not optional. Instead, they are required to be present in every observational data file (see my comment below)